README.rst

```diff
@@ -32,6 +32,26 @@ You can use ``pip`` to install codespell with e.g.::
 
     pip install codespell
 
+You can get shell completion script by::
+
+    pip install "codespell[completion]"
+    codespell --print-completion bash | sudo tee /usr/share/bash-completion/completions/codespell
+    codespell --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_codespell
+    codespell --print-completion tcsh | sudo tee /etc/profile.d/codespell.completion.csh
+
+Or you can get toml support by::
```

Comment:
Pedant, but you could have toml support and shtab, indeed you even have a completion for it:
```suggestion
You can get toml support by::
```

Reply:
I'll fix it.